src/Task/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { AiFillEdit, AiOutlineCheck, AiOutlineClose } from "react-icons/ai";
+import './style.css';
+
+const Task = (props) => {
+
+  // Flag for highlight the card
+  let [classFlag, setClassState] = useState(true);
+  const selectItem = () => setClassState(!classFlag);
+
+  // Stage for type of button and adding input fields
+  let [buttonFlag, setTaskState] = useState(true);
+  const changeText = () => {
+    setClassState(classFlag = true);
+    setTaskState(!buttonFlag);
+  }
+
+  // Intermediate storage facility for text
+  const newTextObj = { ...props.task };
+
+  // Add text from input
+  const newTitle = event => newTextObj.title = event.target.value;
+  // Add text from textarea
+  const newText = event => newTextObj.text = event.target.value;
```

Comment:
These two handlers can still be replaced by the only one handler if pass the property name and set it dynamically.